Editor/SettingsAnalysis/SrpAssetSettingsAnalyzer.cs

```diff
@@ -0,0 +1,146 @@
+using System.Collections.Generic;
+using Unity.ProjectAuditor.Editor.Core;
+using Unity.ProjectAuditor.Editor.Diagnostic;
+using Unity.ProjectAuditor.Editor.Modules;
+using UnityEngine;
+using UnityEngine.Rendering;
+#if PACKAGE_URP
+using UnityEngine.Rendering.Universal;
+#elif PACKAGE_HDRP
+using System.Reflection;
+using UnityEngine.Rendering.HighDefinition;
+#endif
+
+namespace Unity.ProjectAuditor.Editor.SettingsAnalysis
+{
+    class SrpAssetSettingsAnalyzer : ISettingsModuleAnalyzer
+    {
+        internal const string PAS1008 = nameof(PAS1008);
+
+        static readonly Descriptor k_SRPBatcherSettingDescriptor = new Descriptor(
+            PAS1008,
+            "SRP Asset: SRP Batcher",
+            Area.CPU,
+            "SRP batcher is disabled in Render Pipeline Asset.",
+            "Enable SRP batcher in Render Pipeline Asset. This will reduce the CPU time Unity requires to prepare and dispatch draw calls for materials that use the same shader variant.")
+        {
+            messageFormat = "SRP batcher is disabled in '{0}' in '{1}'",
+            fixer = FixSrpBatcherSetting
+        };
+
+        public void Initialize(ProjectAuditorModule module)
+        {
+            module.RegisterDescriptor(k_SRPBatcherSettingDescriptor);
+        }
+
+        public IEnumerable<ProjectIssue> Analyze(ProjectAuditorParams projectAuditorParams)
+        {
+#if UNITY_2019_3_OR_NEWER
+            IEnumerable<ProjectIssue> issues = Analyze(GraphicsSettings.defaultRenderPipeline, -1);
```

Comment:
Just to be sure:

Q: If the QualitySettings for UPR or HDRP are used, should we still check the defaultRenderPipeline here or is it technically irrelevant because the project uses URP / HDRP anyway?

Reply:
QualitySettings may override the defaultRenderPipeline. But if a quality level of the QualitySettings will not override it, the defaultRenderPipeline will be used. So we need to check every RP that may be used: default + RP for each quality level. More info here: https://docs.unity3d.com/Manual/srp-setting-render-pipeline-asset.html

Reply:
Thanks for the explanation!

Sounds good!